### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vaionexbusra**
**Explained Tools**

Hubspot
ShadCn
Vercel CLI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +54,215 @@ The easiest way to deploy your Next.js app is to use the [Vercel Platform](https
...
+
+
+
+## Using ShadCn
+ShadCn provides a streamlined way to add components to your projects. Here's a comprehensive guide on how to use it.
+
+### Command Line Interface (CLI)
+The CLI is your main tool for interacting with ShadCn.
+
+Initializing a New Project
```
Comment (paulsp94) [inline README.md:174]: its already initialized right? can just explain how to add new components

File: README.md [hunk 2]
```diff
@@ -37,7 +54,215 @@ The easiest way to deploy your Next.js app is to use the [Vercel Platform](https
...
+```
+**Project Structure**
+
+Here's a brief on how the project is structured:
+
+**UI Components**: They reside in the components/ui folder.
+
+**Other Components**: Components like <PageHeader /> and <MainNav /> are located in the components directory.
+
+**Utility Functions**: The lib directory holds all utility functions. The cn helper, for instance, is defined in utils.ts.
```
Comment (paulsp94) [inline README.md:221]: dont we have a utils folder? we dont want everything in the same file

File: README.md [hunk 3]
```diff
@@ -37,7 +54,215 @@ The easiest way to deploy your Next.js app is to use the [Vercel Platform](https
...
+
+**Styles**: The styles directory is home to the global CSS.
```
Comment (paulsp94) [inline README.md:223]: should mention that it shouldnt be used, we do everything over tailswind and tailwind theming

**Review by paulsp94 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vaionexbusra) [reply README.md:221]: actually in the utils function we only have createUserCollection.jsx file. and under lib/utils.js. Would you like me to move the utils.js file to under utils file as index.js?

File: README.md [hunk 4]
```diff
@@ -2,27 +2,44 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
+## Setup
+
+### Prerequisites
+
+- Node.js
+- Yarn package manager
+
+### Installation Steps
+
+**1. Clone the VBS repository:**
```
[the remark this answers is not in the document]
Comment (vaionexbusra) [reply README.md:14]: Removed

File: README.md [hunk 5]
```diff
@@ -2,27 +2,44 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+**3. Install the dependencies:**
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+```bash
+yarn install
+```
+```bash
+npm install
```
[the remark this answers is not in the document]
Comment (vaionexbusra) [reply README.md:32]: All npms are removed.

File: README.md [hunk 6]
```diff
@@ -2,27 +2,44 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
+git clone git@github.com:vaionex/VBS.git
 ```
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+**2. Navigate to the VBS directory:**
```
[the remark this answers is not in the document]
Comment (vaionexbusra) [reply README.md:20]: Removed

File: README.md [hunk 7]
```diff
@@ -37,7 +54,215 @@ The easiest way to deploy your Next.js app is to use the [Vercel Platform](https
...
+
+VBS incorporates several popular libraries and tools:
+
+- [Next.js](https://nextjs.org/)
+- [Firebase](https://firebase.google.com/)
+- [HubSpot API Client](https://developers.hubspot.com/)
+- [Radix UI](https://radix-ui.com/)
+- [Tailwind CSS](https://tailwindcss.com/docs/utility-first)
+- [Headless UI](https://headlessui.dev/)
+- [Heroicons](https://heroicons.com/)
```
[the remark this answers is not in the document]
Comment (vaionexbusra) [reply README.md:67]: removed. also gonna update the related pages in the futher PRs.

File: README.md [hunk 8]
```diff
@@ -37,7 +54,215 @@ The easiest way to deploy your Next.js app is to use the [Vercel Platform](https
...
+## Usage
+
+### Local Development
+
+To run the VBS project locally:
+
+```bash
+yarn dev
+```
+```bash
```
[the remark this answers is not in the document]
Comment (vaionexbusra) [reply README.md:83]: removed.

File: README.md (hunk 1 above)
Comment (vaionexbusra) [reply README.md:174]: Initialization part removed.

File: README.md (hunk 2 above)
Comment (paulsp94) [reply README.md:175]: yeah not even as index but just as common or something like that

File: README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (paulsp94) [reply README.md]: did you also remove it from the package json?

[the remark this answers is not in the document]
Comment (vaionexbusra) [reply README.md]: ı haven't removed the heroicons/lucide  fro the components. Would you like me to in this PR?

**Review by paulsp94 — APPROVED** [review]
LGTM

**Pull request merged**